src/main/java/Duke.java

```diff
@@ -1,10 +1,119 @@
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    public static void printList(Task[] l1, int currListIndex) {
+        int index;
+        if (l1[0] == null) {
```

Comment:
I notice that you have a tendency to use magic strings and numbers. Do try to avoid that where possible, and instead use constants to set your strings/numbers.

Reply:
Okie! I will go and create a constants file for messages and numbers to fix this! Thank you!